src/RemoteApi/Actions/GetBalances/Action.php

```diff
@@ -72,7 +75,7 @@ public function setAddresses(array $addresses): self
      */
     public function addAddress(Address $address): self
     {
-        $this->addresses[] = $address->removeChecksum();
```

Comment:
Thats not correct, if someone calls addAddress with an address including the checksum the checksum is not removed.

Reply:
Now it's removing it twice when you call `setAddresses`